app/src/views/AllAppeals/index.tsx

```diff
@@ -137,11 +137,19 @@ export function Component() {
         (country) => country,
     );
 
+    let betterOrdering = ordering;
+    if (ordering === '-id') {
+        betterOrdering = '-start_date';
+    } else if (ordering.replace('-', '') !== 'start_date') {
+        // eslint-disable-next-line
+        betterOrdering = ordering + ',-start_date';
```

Comment:
Why do we need to disable eslint here? Does it just not like `+` to concatenate? In that case, maybe ``${ordering},-start_date`` fixes that?

Reply:
Nice one!
Thanks @batpad !

Reply:
Unfortunately this syntax does not work. Only a backtick `` -like syntax would work, which is much uglier than the simple +. Or, if you can suggest me a +less concatenation, let me know.